runtime/src/openxla/runtime/nvgpu/cudnn/test/e2e_conv2d.mlir

```diff
@@ -0,0 +1,57 @@
+// RUN: iree-compile %s --mlir-print-ir-after-all --iree-plugin=openxla_nvgpu  \
```

Comment:
Maybe print only after lowering to vm? Should help with debugging and should make the test faster.

Reply:
Thanks!